lib/drivers/ripple.js

```diff
@@ -11,11 +11,11 @@ const Supply = require('../models/supply');
 class Ripple extends Driver {
   constructor(options) {
     super({
-      timeout: 100, // 10 requiest per second
+      timeout: 100, // 10 request per second
       supports: {
         circulating: true,
         max: true,
-        blockchains: ['XRP'],
+        blockchains: ['Ripple'],
```

Comment:
A bit hard to find out what the real name of the blockchain (network) is, but I think it is best explained on https://xrpl.org/ and in the repository https://github.com/ripple/rippled
```suggestion
        blockchains: ['XRP Ledger'],
```